R/download.input.R

```diff
@@ -0,0 +1,65 @@
+##' Downloads & saves the desired input file.
+##' Hits the `/api/inputs/{input_id}` API endpoint
+##' @name download.input
+##' @title Download & save the desired input file
+##' @param server Server object obtained using the connect() function
+##' @param input_id ID of the PEcAn input to be downloaded
+##' @param filename Optional filename specified if the id points to a folder instead of file
+##' @param save_as File name to save the downloaded file as. Default: "pecan_input_file"
+##' @return Response obtained from the `/api/inputs/{input_id}` endpoint
+##' @author Tezan Sahu
+##' @export
+##' @examples
+##' server <- connect(url="http://localhost:8000", username="carya", password="illinois")
+##' 
+##' # Download the 'niwot.clim' file (id = 99000000003)
+##' download.input(server, input_id='99000000003', save_as='local.niwot.clim')
+
+download.input <- function(server, input_id, filename=NULL, save_as="pecan_input_file"){
```

Comment:
seems like the `save_as` should default to `filename` unless filename is NULL

Reply:
Sure...and what would we do if `filename` is null, & `save_as` is also null? I guess for that case we should have some default name...right?

Reply:
is there some way of putting an ifelse in the default? e.g. save_as = ifelse(is.null(filename),"pecan_input_file",filename)

Reply:
Yeah...I can try doing that

Reply:
This suggestion has been addressed in my latest commit d20b7af